src/components/SignIn/index.js

```diff
@@ -12,6 +12,12 @@ function SignIn() {
   const [formData, setFormData] = useState(initialState)
   const [error, setError] = useState(false)
 
+  useEffect(() => {
+    const userId = localStorage.getItem('userId')
+    const token = localStorage.getItem('token')
+    if (userId && token) navigate('/')
+  }, [navigate])
```

Comment:
We should create a function that determines whether or not a user has a `token` and `userId` by returning a boolean value that can be used in conditional statements.

Reply:
We can introduce this in a different PR.

Reply:
https://github.com/enickgoku/facerecognitionbrain/issues/2